src/sentry/api/helpers/group_index/update.py

```diff
@@ -715,7 +720,7 @@ def handle_other_status_updates(
                 )
             else:
                 result["statusDetails"] = handle_ignored(
-                    group_ids, group_list, status_details, acting_user, user
+                    group_list, dict(status_details), acting_user, user
```

Comment:
I suspect that rather than copying the dict here the types should change from `Mapping` to `dict`